### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by alanvardy**
Hi Matt!

Just updating the dependencies on my website and got blocked by the ecto requirements.

Lets chat soon! :)

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -33,7 +33,7 @@ defmodule ExEffectiveBootstrap.MixProject do
       {:phoenix_html, "~> 2.1"},
       {:jason, "~> 1.2"},
       {:plug_cowboy, "~> 2.0"},
-      {:ecto, "~> 3.5"},
+      {:ecto, "~> 3.6"},
```
Comment (alanvardy) [inline mix.exs:36]: The "offending" dep

File: .tool-versions [hunk 2]
```diff
@@ -0,0 +1,2 @@
+elixir 1.12.1
+erlang 24.0
```
Comment (alanvardy) [inline .tool-versions:2]: This is pretty arbitrary, just set it to the latest versions.

File: mix.lock [hunk 3]
```diff
@@ -1,25 +1,30 @@
...
+  "plug_cowboy": {:hex, :plug_cowboy, "2.5.0", "51c998f788c4e68fc9f947a5eba8c215fbb1d63a520f7604134cab0270ea6513", [:mix], [{:cowboy, "~> 2.7", [hex: :cowboy, repo: "hexpm", optional: false]}, {:cowboy_telemetry, "~> 0.3", [hex: :cowboy_telemetry, repo: "hexpm", optional: false]}, {:plug, "~> 1.7", [hex: :plug, repo: "hexpm", optional: false]}, {:telemetry, "~> 0.4", [hex: :telemetry, repo: "hexpm", optional: false]}], "hexpm", "5b2c8925a5e2587446f33810a58c01e66b3c345652eeec809b76ba007acde71a"},
+  "plug_crypto": {:hex, :plug_crypto, "1.2.2", "05654514ac717ff3a1843204b424477d9e60c143406aa94daf2274fdd280794d", [:mix], [], "hexpm", "87631c7ad914a5a445f0a3809f99b079113ae4ed4b867348dd9eec288cecb6db"},
+  "ranch": {:hex, :ranch, "1.8.0", "8c7a100a139fd57f17327b6413e4167ac559fbc04ca7448e9be9057311597a1d", [:make, :rebar3], [], "hexpm", "49fbcfd3682fab1f5d109351b61257676da1a2fdbe295904176d5e521a2ddfe5"},
+  "telemetry": {:hex, :telemetry, "0.4.3", "a06428a514bdbc63293cd9a6263aad00ddeb66f608163bdec7c8995784080818", [:rebar3], [], "hexpm", "eb72b8365ffda5bed68a620d1da88525e326cb82a75ee61354fc24b844768041"},
```
Comment (alanvardy) [inline mix.lock:29]: Updated the rest of the things in the last commit.

**Comment by matt-riemer** [comment]
Thanks!!

**Pull request merged**